code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -82,6 +82,7 @@
 /obj/item/ego_weapon/eyeball
 	name = "eyeball scooper"
 	desc = "Mind if I take them?"
+	special = "This weapon grows more powerful as you do, but its potential is limited if you aren't loyal to it."
```

Comment:
While this is a very nice description, it's not a good one for saying how it actually works.
```suggestion
	special = "This weapon grows more powerful as you do, but its potential is limited if you possess any other EGO weapons."
```

Reply:
Optionally,
```suggestion
	special = "This weapon grows more powerful as you do, but its potential is limited if you aren't loyal to it, and are possessing other EGO weapons."
```